app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -287,10 +288,13 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        if (!SupportedSnapdragonBoards.isBoardSupported(SystemInfoUtils.getSnpeSupportedBoard())) {
+        val board = SystemInfoUtils.getSnpeSupportedBoard()
+
+        if (!SupportedSnapdragonBoards.isBoardSupported(board)) {
             val text =
                 Html.fromHtml("The device is not supported, you need <b>Snapdragon-powered</b> device with <b>OpenCL</b> support, more details at <b>https://www.mapbox.com/android-docs/vision/overview/</b>")
             Toast.makeText(this, text, Toast.LENGTH_LONG).show()
+            VisionLogger.e("NotSupportedBoard", "Current board is {\"$board\"}")
```

Comment:
M, why do we need the log in logcat? Will it help us or users somehow?

Reply:
It helps to us get name of current `Board` if it doesn't work